### DISCUSSION TITLE
Student, but Copilot can't use

### DISCUSSION THREAD

**Discussion by HurmousDay** [community/community#114065] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I passed my student accreditation and it's valid until May '24, why does copilot now say it's expired and can't be used?

![image](https://github.com/community/community/assets/44956408/bae541cd-a4cb-4269-958f-15efdf22de62)
![image](https://github.com/community/community/assets/44956408/9015858b-a269-4374-9c45-b189eee74e2f)

**Comment by VegaCaroMiguel** [comment]


**Reply by HurmousDay** [reply]
Hello ！ I've updated copilot to the latest version and restarted the IDE, but copilot still shows a problem

![image](https://github.com/community/community/assets/44956408/52e716cf-8016-43d1-9b4f-cb11cefdb9b4)

**Reply by HurmousDay** [reply]
Hi, I'm on v1.175.0, which should be the latest version, as shown in the picture, and have restarted vscode with the plugin, to no avail!

<img width="685" alt="image" src="https://github.com/community/community/assets/44956408/9cc5a18d-68b3-4af3-ab0a-537121f0b3a1">

![image](https://github.com/community/community/assets/44956408/496e3fd7-df62-4c71-8122-4c995e13e93e)

**Reply by HurmousDay** [reply]
Hi, I tried with other IDEs in Jetbrain (uninstalling and installing) and the result is the same, it says I don't have permissions

![image](https://github.com/community/community/assets/44956408/8ab6cd7e-09d6-4911-8ab7-ee786d22a9cd)
![image](https://github.com/community/community/assets/44956408/1d189b75-0107-4036-80f9-5cdd2e6fcb43)

**Reply by HurmousDay** [reply]
And I was able to open this page, I should have student access, web access to github is pretty fast
<img width="1476" alt="image" src="https://github.com/community/community/assets/44956408/e7f5f0e6-5946-4d9a-ab6a-e67382e9dfcc">

**Reply by HurmousDay** [reply]
Hello, It shouldn't have anything to do with authentication, because I've gone through the process of authentication in Goland, and the final error was that I hadn't opened copilot
<img width="598" alt="image" src="https://github.com/community/community/assets/44956408/4d092107-08b5-4e9e-9b24-8191ebb2e2d0">
<img width="440" alt="image" src="https://github.com/community/community/assets/44956408/a1075121-553f-42ce-ba5e-21dedfe442f7">

**Comment by davecheney** [comment, marked as answer]
@HurmousDay 
We cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
